mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -2,6 +2,9 @@ CLASS ail net/minecraft/entity/Entity
 	FIELD A velocityModified Z
 	FIELD B movementMultiplier Lcsa;
 	FIELD C removed Z
+	FIELD D previousHorizontalSpeed F
```

Comment:
We use `last` rather than `previous` for the value last tick (see WorldRenderer, for example).

Reply:
`last` is being used here for the `lastPortal` value. I will shorten it to `prevHorizontalSpeed` though to match the other `prev` values in this class.

Reply:
Generally, I dislike the use of `last` itself, anyway, as it doesn't properly convey the meaning that it is the _previous_ value. With `previous` you know there must be a _current_ counterpart, but last doesn't have that assumption. `last` just says it is the value as it was last known.